src/libsyntax/parse/lexer/mod.rs

```diff
@@ -1455,8 +1456,9 @@ impl<'a> StringReader<'a> {
                         }
```

Comment:
I believe that this check is supposed to catch the `let x = 'asd // '` case, but it isn't... I wonder what's going on.